### PR TITLE
Update to Moshi 1.5.0 and OkHttp 3.8.1.

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

**Comment by digitalbuddha** [comment]
Lgtm

**Comment by NightlyNexus** [comment]
oops, source incompatible. will check in a minute.

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 def versions = [
     apolloVersion                : '0.3.2-SNAPSHOT',
     cacheVersion                 : '2.0.2',
-    okHttpVersion                : '3.6.0',
+    okHttpVersion                : '3.8.1',
```
Comment (NightlyNexus) [inline gradle/dependencies.gradle]: alternatively, i can explicitly require Okio 1.13.0 if there's a reason to stay on OkHttp 3.6.0.

**Comment by digitalbuddha** [comment]
#501 react native does some okhttp dependency tomfoolery. Not sure if ticking the version would cause this problem again.

**Comment by BenSchwab** [comment]
`3.8.0` had some crashes in react native. Presumably `3.8.1` fixes those crashes.

**Pull request merged**